packages/worker/src/index.ts

```diff
@@ -308,9 +296,9 @@ const pollTranscriptionQueue = async (
 			taskMessage.Attributes?.ApproximateReceiveCount || defaultReceiveCount,
 		);
 		if (receiveCount >= MAX_RECEIVE_COUNT) {
-			publishTranscriptionOutputFailure(
-				snsClient,
-				config.app.destinationTopicArns.transcriptionService,
+			await publishTranscriptionOutputFailure(
```

Comment:
good call - it's probably important to await the result of this before updating scale in protection